src/main/java/com/gaggle/contact/domain/Contact.java

```diff
@@ -14,11 +14,13 @@
 
     private @Id @GeneratedValue Long id;
     private @NotBlank String name;
+    private String email;
 
     public Contact() {}
 
-    public Contact(final String name) {
+    public Contact(final String name, final String email) {
```

Comment:
We're using a private variables to reference values from arguments that are final; let's make them all private or all final.